File: framework/web/src/test/java/cn/bestwu/simpleframework/support/setting/PropertiesSourceTest.java
package cn.bestwu.simpleframework.support.setting;

import cn.bestwu.lang.property.PropertiesSource;
import java.util.Map;
import org.junit.jupiter.api.Test;

/**
 * @author Peter Wu
 */
class PropertiesSourceTest {

  @Test
  void mapOf() {
    PropertiesSource propertiesSource = new PropertiesSource("base-messages");
    Map<Object, Object> typeMismatch = propertiesSource.mapOf("typeMismatch");
    typeMismatch.forEach((k, v) -> System.err.println(k + ":" + v));
  }

  @Test
  void test() {
    PropertiesSource propertiesSource = new PropertiesSource("default-exception-handle");
    propertiesSource.all().forEach((k, v) -> System.err.println(k + ":" + v));
  }
}

File: web/src/test/kotlin/top/bettercode/summer/test/web/support/setting/PropertiesSourceTest.kt
package top.bettercode.summer.test.web.support.setting

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import top.bettercode.summer.tools.lang.property.PropertiesSource

/**
 * @author Peter Wu
 */
internal class PropertiesSourceTest {
    @Test
    fun mapOf() {
        val propertiesSource = PropertiesSource("base-messages")
        val typeMismatch = propertiesSource.mapOf("typeMismatch")
        Assertions.assertTrue(typeMismatch.isNotEmpty())
//        typeMismatch.forEach { (k: String, v: String) -> System.err.println("$k:$v") }
    }

    @Test
    fun test() {
        val propertiesSource = PropertiesSource("default-exception-handle")
        Assertions.assertTrue(propertiesSource.all().isNotEmpty())
//        propertiesSource.all().forEach { (k: String, v: String) -> System.err.println("$k:$v") }
    }
}